resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -748,6 +756,18 @@ export default {
             }
         }
 
+        Events.$on('focusTypeaheadEvent', () => { 
+            this.focusInputFindlabel = false;
+            this.$nextTick(() => {
+                this.focusInputFindlabel = true;
+            });
+           this.focusInputFindlabel = false;
```

Comment:
```suggestion
```
I don't think we need it here any more, because you set `this.focusInputFindlabel` on false in line 760 already. But maybe there is a reason why you need both?